htdocs/mri/jiv/get_file.php

```diff
@@ -60,13 +60,6 @@
 // Now get the file and do file validation
 $File = $_GET['file'];
 
-// File validation
-if (strpos($File, ".") === false) {
-    error_log("ERROR: Could not determine file type.");
-    header("HTTP/1.1 400 Bad Request");
-    exit(3);
-}
-
```

Comment:
We can... I just find it a little odd in the context of this script. If we actually wanna do filetype checking we can use `filetype`. If no `.` is specified the logic should fall through that this is a `404` which is also correct.